src/styles/searchResults.css

```diff
@@ -0,0 +1,17 @@
+.search-results {
+  margin: 10px;
+  border-radius: 4px;
+  padding: 5px;
+  width: 220px;
+}
+
+img > .search-results {
+  width: 5px;
+  height: 5px;
+}
```

Comment:
this css isn't targeting anything and can be removed. It is trying to target an element with a class of `search-results` that is a child of an `img`, but your `img` has the class of `.search-results` on it